hack/dockerfile_install_support.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -euo pipefail
+set -o xtrace
+
+source /etc/os-release
+if [[ "${ID}" == "centos" ]]; then
```

Comment:
Maybe this should be centos OR fedora, in case we bring back OKD/FCOS builds

Reply:
actually, nevermind, this wouldn't work, as OKD/FCOS payload images would still be using the centos builds.

Maybe this should instead make use of the `ARG TAGS=ocp`, which would be overriden in the OKD builds with TAGS=scos for OKD/SCOS and TAGS=fcos for OKD/FCOS.

Reply:
Anyway, not a blocker.